lang/src/lib.rs

```diff
@@ -291,12 +289,9 @@ pub mod __private {
 
     use solana_program::pubkey::Pubkey;
 
-    pub mod state {
-        pub use crate::accounts::state::*;
-    }
-
     // Calculates the size of an account, which may be larger than the deserialized
-    // data in it. This trait is currently only used for `#[state]` accounts.
+    // data in it. This trait is currently unused.
+    // TODO: Uses it maybe to calculate the size of an account.
```

Comment:
minor english `maybe use it calculate ...`

The key with making a trait that automatically calculates size is that it doesn't have a high runtime cost. It really ought to be closer to a constant, although need to account for variable sized elements.